docs/versions/latest/Release.md

```diff
@@ -26,6 +26,21 @@
 
 
 
+### Implemented Push/Pull logic for new Content Sharing Policy from vRA 8.8.2
```

Comment:
These file changes are not relevant for this PR. Please include only information about creating backups (most probably this happened because we had a release yesterday)